src/shogun/machine/LinearMachine.cpp

```diff
@@ -12,11 +12,19 @@
 #include <shogun/labels/RegressionLabels.h>
 #include <shogun/features/DotFeatures.h>
 #include <shogun/labels/Labels.h>
+#include <shogun/mathematics/eigen3.h>
 
 using namespace shogun;
+using namespace Eigen;
 
 CLinearMachine::CLinearMachine()
-: CMachine(), bias(0), features(NULL)
+: CMachine(), bias(0), features(NULL),bias_flag(true)
```

Comment:
These constructor initialisations have caused troubles in the past.
Can you

 * Initialise all members to default values in the init method (before SG_ADD)
 * call init from constructor
 * overwrite default values the were passed as parameters